src/CMakeLists.txt

```
@@ -1,5 +1,6 @@
 add_subdirectory(realm)
 add_subdirectory(external/IntelRDFPMathLib20U2)
+add_subdirectory(external/bson)
```

```suggestion
add_subdirectory(external/bson EXCLUDE_FROM_ALL)
```

This will exclude any targets defined in the `external/bson` listfile from the default `ALL` target so it's not built needlessly. Or do we want to be building the new target always in the mean time even if nothing else is linking against it?

---

I am not sure it makes a difference. The library is needed by realm-tests which is included in ALL.

---

The tests aren’t built if REALM_BUILD_LIB_ONLY or REALM_NO_TESTS are specified or if realm-core is a subproject of another CMake project.